### ISSUE TITLE
FR: split view in portrait orientation

### ISSUE THREAD

**Issue by Brynjulf** [wix/react-native-navigation#4450]
### Issue Description

I could not find any documentation to show split view while in portrait orientation

### Steps to Reproduce / Code Snippets / Screenshots

```
const splitView = {
  id: 'splitView1',
  master: {
    stack: {
      id: 'masterStack',
      children: [
        {
          component: {
            name: 'FeedScreen',
            options: {
              topBar: {
                visible: true,
                title: {
                  text: 'master',
                }
              }
            }
          },
        },
      ],
    },
  },
  detail: {
    stack: {
      id: 'detailStack',
      children: [
        {
          component: {
            name: 'HomeScreen',
            options: {
              topBar: {
                title: {
                  visible: false,
                  text: 'detail',
                }
              }
            }
          },
        },
      ],
    },
  },
  options: {
    displayMode: 'visible', // Master view display mode: `auto`, `visible`, `hidden` and `overlay`
    primaryEdge: 'leading', // Master view side: `leading` or `trailing`
    minWidth: 150, // Minimum width of master view
    maxWidth: 300, // Maximum width of master view
  },
  
};
```
---
### Environment
* React Native Navigation version: 2.1.3-patch.2
* React Native version: 0.57.8
* Platform(s) (iOS, Android, or both?): iOs
* Device info (Simulator/Device? OS version? Debug/Release?): Simulator 10.1.14 Debug

**Comment by PSPanesar** [comment]
Hi,

I've had the same problem in the in portrait mode i can't find a way of showing the master view first. The options applied to the splitview don't seem to make any difference regarding displayMode. 

Have you found a solution to this yet?

In a native iOS appt there would be a function that you can override re collapse primary on secondary but i'm not sure how you do it in react native navigation. Essentially what i am trying to do is have the master open first in portrait so you can see the table before the detail view.

Thanks

**Comment by zzorba** [comment]
So I'm seeing a somewhat what might be a related issue, but the top level 'options' for splitView such as `displayMode` and `primaryEdge` don't seem to have any effect on the SplitView.  It's always on the left, and its always visible.

I even tried 'merging' the options in to hide it on mount, but still no luck.

**Comment by Brynjulf** [comment]
@PSPanesar, I saw a (SO post)[https://stackoverflow.com/questions/4790804/split-view-in-portrait-mode] about a way to disable hiding details view. Although it is an old solution for old os', i was hoping to have this feature in RNN as an option.

@zzorba , you are correct, i left those options in there because this was a copy from some example. seems like all you need is to declare `splitview` with a detail and a master

**Comment by zzorba** [comment]
@PSPanesar @Brynjulf, I dug deep into the source code this weekend and managed to get the splitView options working again! Created a PR here -- https://github.com/wix/react-native-navigation/pull/4647/files hopefully it will get reviewed and merged soon.

If you'd like to help me validate this change, you can set your package.json for 'react-native-navigation' to `"zzorba/react-native-navigation#fix_splitView_options"` to try it out. Please let me know if you encounter any issues with it.

Note, in a previous change to make the 'none' splitView specific options work, I moved the `splitView` options down one level, so your options should now look like this when using a SplitView. That change got released in 2.8.0
```
options: {
  splitView: {
    displayMode: 'always',
  },
  // other options here, like bottomTab
}
```
